apps/chat/src/components/Chatbar/Conversation.tsx

```diff
@@ -120,13 +120,20 @@ export function ConversationView({
         data-qa="conversation-name"
       >
         <Tooltip
-          tooltip={t(
-            getEntityNameError(isNameInvalid, isInvalidPath, isExternal),
-          )}
-          hideTooltip={!isNameOrPathInvalid}
-          triggerClassName="block max-h-5 flex-1 truncate whitespace-pre break-all text-left"
+          tooltip={
+            conversation.name ||
+            t(getEntityNameError(isNameInvalid, isInvalidPath, isExternal))
```

Comment:
in this case error never will be visible
```suggestion
            t(getEntityNameError(isNameInvalid, isInvalidPath, isExternal))
            ||  conversation.name
```

Reply:
Thanks for your comment, I fixed it.